### DISCUSSION TITLE
Customize `make:filament-resource` templates

### DISCUSSION THREAD

**Discussion by faydanube** [filamentphp/filament#11632] [Ideas]
Filament doesn't seem to provide an official way to publish the stubs of its resource generator directly via commands, I've carefully searched the official documentation, and I'm sorry if I'm missing something.

If the current version doesn't support it, I hope the next version can publish stubs for users to customize, thanks a lot!

**Comment by SydVic** [comment]
I was searching the same thing and found this. Hope that can help.
https://filamentphp.com/docs/3.x/support/stubs
